release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2101,6 +2101,10 @@ Workaround: Manually or by using a daemon set, modify the file `/etc/cni/tuning/
 
 * For {product-title} {product-version}, the Driver Toolkit (DTK) container image requires the `ubi9` image as the second layer of the software stack for building driver containers. If you attempt to use the `ubi8` image as the second layer in the your software stack, you will receive a build error. (link:https://issues.redhat.com/browse/OCPBUGS-11120[*OCPBUGS-11120*])
 
+* In some {product-title} installations on vSphere platform when using CSI driver, the vSphere CSI driver may not come up correctly because on startup it fails to retrieve information about a node from vCenter and CSI driver does not retry.
```

Comment:
Recommend:

In some {product-title} installations on the vSphere platform when using the CSI driver, the vSphere CSI driver might not come up correctly because during startup it fails to retrieve information about a node from vCenter, and then the CSI driver does not retry.
+
Workaround: By using SSH to contact the node, which is current leader of the vsphere-syncer process, and restarting the vsphere-syncer container (using `crictl`), this issue can be mitigated and the driver successfully comes up. (link:https://issues.redhat.com/browse/OCPBUGS-13385[*OCPBUGS-13385*])

Reply:
Incorporated.